src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -0,0 +1,89 @@
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.title'|trans({}, 'locationview') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.creator'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.lastcontributor'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.versions'|trans({}, 'locationview') }}</th>
```

Comment:
if the string is different in English, the purpose is not really the same, 'Versions' and 'Published version' does not mean the same thing at all. So for me, it's clear we should use different key than the one we were using in v1.